test/test_prototype_builtin_datasets.py

```diff
@@ -104,13 +112,13 @@ def test_transformable(self, dataset_mock, config):
     @pytest.mark.parametrize("only_datapipe", [False, True])
     @parametrize_dataset_mocks(DATASET_MOCKS)
     def test_traversable(self, dataset_mock, config, only_datapipe):
-        dataset, _ = dataset_mock.load(config)
+        dataset = dataset_mock.load(config)[0].map(self._TRANSFORMS)
```

Comment:
Could you do this in two lines, to have the same loading call on all tests?

```suggestion
        dataset, _ = dataset_mock.load(config)
        dataset = dataset.map(self._TRANSFORMS)
```